src/Core/OrganizationFeatures/OrganizationSponsorships/FamiliesForEnterprise/Cloud/CloudRevokeSponsorshipCommand.cs

```diff
@@ -3,39 +3,32 @@
 using Bit.Core.Exceptions;
 using Bit.Core.OrganizationFeatures.OrganizationSponsorships.FamiliesForEnterprise.Interfaces;
 using Bit.Core.Repositories;
-using Bit.Core.Services;
 
 namespace Bit.Core.OrganizationFeatures.OrganizationSponsorships.FamiliesForEnterprise.Cloud
 {
-    public class CloudRevokeSponsorshipCommand : CloudCancelSponsorshipCommand, ICloudRevokeSponsorshipCommand
+    public class CloudRevokeSponsorshipCommand : CancelSponsorshipCommand, IRevokeSponsorshipCommand
     {
         public CloudRevokeSponsorshipCommand(
             IOrganizationSponsorshipRepository organizationSponsorshipRepository,
-            IOrganizationRepository organizationRepository,
-            IPaymentService paymentService,
-            IMailService mailService) : base(organizationSponsorshipRepository, organizationRepository, paymentService, mailService)
+            IOrganizationRepository organizationRepository) : base(organizationSponsorshipRepository, organizationRepository)
         {
         }
 
-        public async Task RevokeSponsorshipAsync(Organization sponsoredOrg, OrganizationSponsorship sponsorship)
+        public async Task RevokeSponsorshipAsync(OrganizationSponsorship sponsorship)
         {
             if (sponsorship == null)
             {
                 throw new BadRequestException("You are not currently sponsoring an organization.");
             }
 
-            if (sponsorship.SponsoredOrganizationId == null)
+            if (sponsorship.SponsoredOrganizationId == default)
```

Comment:
I prefer `null` here even though default would work just fine.

Reply:
oops misread this as `Sponsor**ing**` I agree.